src/css/quasar.variables.sass

```diff
@@ -89,3 +89,14 @@ body
     background-color: var(--bg-color)
 
 
+// slide-in-left animation
+$anim-slide-in-left: slide-in-left 0.3s cubic-bezier(0,1.02,1,.99) both
```

Comment:
One huge improvement would be to have `slide-out-right` and apply it when clicking the back button. What do you think?
We can, of course, address this in another PR/issue.

Reply:
Of course, but unfortunately, it wouldn't be that simple. The back button should emit two signals separated in time. The first one to trigger the exit animation and the other to actually go back. This is not one line of code, so I created a [new issue](https://github.com/telosnetwork/telos-wallet/issues/272) to address this in the future.